atom/common/lib/asar.coffee

```diff
@@ -68,10 +77,10 @@ overrideAPISync = (module, name, arg = 0) ->
     return old.apply this, arguments unless isAsar
 
     archive = getOrCreateArchive asarPath
-    throw new Error("Invalid package #{asarPath}") unless archive
+    invalidArchiveError asarPath unless archive
 
     newPath = archive.copyFileOut filePath
-    throw createNotFoundError(asarPath, filePath) unless newPath
+    createNotFoundError(asarPath, filePath) unless newPath
```

Comment:
Since the `throw` expression becomes a function call, we should remove the braces in it: `createNotFoundError asarPath, filePath unless newPath`, so it matches the style of other code.